docs/examples/kops-test-route53-subdomain.md

```diff
@@ -0,0 +1,870 @@
+|+----------------------------------------------------+----------------+-------------+|
+||  kopsclustertest.kopeio.org.                       |  172800        |  NS         ||
+|+----------------------------------------------------+----------------+-------------+|
+|||                                 ResourceRecords                                 |||
+||+---------------------------------------------------------------------------------+||
+|||                                      Value                                      |||
+||+---------------------------------------------------------------------------------+||
+|||  ns-1383.awsdns-44.org.                                                         |||
+|||  ns-829.awsdns-39.net.                                                          |||
+|||  ns-346.awsdns-43.com.                                                          |||
+|||  ns-1973.awsdns-54.co.uk.                                                       |||
+||+---------------------------------------------------------------------------------+||
+||                                ResourceRecordSets                                 ||
+|+-------------------------------------------------------+------------+--------------+|
+||                         Name                          |    TTL     |    Type      ||
+|+-------------------------------------------------------+------------+--------------+|
+||  kopsclustertest.kopeio.org.                          |  900       |  SOA         ||
+|+-------------------------------------------------------+------------+--------------+|
+|||                                 ResourceRecords                                 |||
+||+---------------------------------------------------------------------------------+||
+|||                                      Value                                      |||
+||+---------------------------------------------------------------------------------+||
+|||  ns-1383.awsdns-44.org. awsdns-hostmaster.amazon.com. 1 7200 900 1209600 86400  |||
+||+---------------------------------------------------------------------------------+||
+```
+
+Also, do a "dig" test in order to check the zone availability on the Internet:
+
+```bash
+dig +short kopsclustertest.kopeio.org soa
+
+ns-1383.awsdns-44.org. awsdns-hostmaster.amazon.com. 1 7200 900 1209600 86400
+
+dig +short kopsclustertest.kopeio.org ns
+
+ns-1383.awsdns-44.org.
+ns-829.awsdns-39.net.
+ns-1973.awsdns-54.co.uk.
+ns-346.awsdns-43.com.
+```
+
+If both your SOA and NS records are there, then your subdomain is ready to be used by KOPS.
+
+
+## AWS/KOPS ENVIRONMENT INFORMATION SETUP:
+
+First, using some scripting and asuming you already configured your "aws" environment on your linux system, use the following commands in order to export your AWS access/secret (this will work if you are using the default profile):
+
+```bash
+export AWS_ACCESS_KEY_ID=`grep aws_access_key_id ~/.aws/credentials|awk '{print $3}'`
```

Comment:
Why are we exporting our IDs? Should not have to

Reply:
asuming --> assuming

Reply:
The guide here: https://github.com/kubernetes/kops/blob/master/docs/aws.md states:
"# Because "aws configure" doesn't export these vars for kops to use, we export them now
export AWS_ACCESS_KEY_ID=<access key>
export AWS_SECRET_ACCESS_KEY=<secret key>
"
If you can confirm this is no longer needed, I'll get rid of this on the documents.